index.js

```diff
@@ -0,0 +1,6 @@
+/* jshint node: true */
+'use strict';
+
+module.exports = {
+  name: 'model-fragments'
```

Comment:
I have a feeling this will interfere with installing the module itself. Need to test it out with `npm link`.